gr-filter/python/filter/bindings/firdes_python.cc

```diff
@@ -49,7 +49,7 @@ void bind_firdes(py::module& m)
                     py::arg("cutoff_freq"),
                     py::arg("transition_width"),
                     py::arg("window") = ::gr::fft::window::win_type::WIN_HAMMING,
-                    py::arg("beta") = 6.7599999999999998,
+                    py::arg("param") = 6.7599999999999998,
```

Comment:
Should the truncated values (which were automatically added at some point) be changed here to the intended 6.76?

Reply:
I thought about doing that....but figured autogenerated code might get autogenerated again and it's pretty clear where the value came from.